torchrec/test_utils/__init__.py

```diff
@@ -22,7 +23,7 @@
 
 TParams = ParameterSpecification("TParams")
 TReturn = TypeVar("TReturn")
-
+TEST_WITH_ROCM: bool = os.getenv("TORCHREC_TEST_WITH_ROCM", "0") == "1"
```

Comment:
The default value of this is 0. So, the CI script needs this env variable set.

Reply:
@amathews-amd I updated the CI script https://github.com/ROCmSoftwarePlatform/rocAutomation/blob/pytorch_torchrec_ci/pytorch/torchrec_ci/build_test_torchrec.sh#:~:text=Run%20torchrec%20unittests-,TORCHREC_TEST_WITH_ROCM%3D1,-pytest%20torchrec/datasets

Reply:
CI seems to have failed